test/solidity/compiler/index.ts

```diff
@@ -66,17 +66,22 @@ contract A {}
       downloader
     );
 
-    const output = await compiler.compile(input);
-
-    // We just check some properties
-    assert.isDefined(output.contracts);
-    assert.isDefined(output.contracts["A.sol"]);
-    assert.isDefined(output.contracts["A.sol"].A);
-
-    assert.isDefined(output.sources);
-    assert.isDefined(output.sources["A.sol"]);
-    assert.isDefined(output.sources["A.sol"].ast);
-    assert.equal(output.sources["A.sol"].id, 0);
+    compiler
+      .compile(input)
+      .then(output => {
+        // We just check some properties
+        assert.isDefined(output.contracts);
+        assert.isDefined(output.contracts["A.sol"]);
+        assert.isDefined(output.contracts["A.sol"].A);
+
+        assert.isDefined(output.sources);
+        assert.isDefined(output.sources["A.sol"]);
+        assert.isDefined(output.sources["A.sol"].ast);
+        assert.equal(output.sources["A.sol"].id, 0);
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
This catches all errors and won't make the CI fail if we break something in the future. Using `await` had that intended behavior